In LC_IDENTIFICATION, audience, application and abbreviation keywords
are optional, thus do not report an error if they are not defined.

# DP: Dpatch author: Denis Barbier
# DP: Patch author: Denis Barbier
# DP: Upstream status: not submitted
# DP: Date: 2006-01-08

---
 locale/programs/ld-identification.c |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

Index: glibc-2.28/locale/programs/ld-identification.c
===================================================================
--- glibc-2.28.orig/locale/programs/ld-identification.c
+++ glibc-2.28/locale/programs/ld-identification.c
@@ -145,6 +145,9 @@ No definition for %s category found"), "
 			#cat);						      \
       identification->cat = "";						      \
     }
+#define TEST_ELEM_OPT(cat)						      \
+  if (identification->cat == NULL)					      \
+    identification->cat = "";						      \
 
   TEST_ELEM (title);
   TEST_ELEM (source);
@@ -155,9 +158,9 @@ No definition for %s category found"), "
   TEST_ELEM (fax);
   TEST_ELEM (language);
   TEST_ELEM (territory);
-  TEST_ELEM (audience);
-  TEST_ELEM (application);
-  TEST_ELEM (abbreviation);
+  TEST_ELEM_OPT (audience);
+  TEST_ELEM_OPT (application);
+  TEST_ELEM_OPT (abbreviation);
   TEST_ELEM (revision);
   TEST_ELEM (date);
 
